app/utils.py

```diff
@@ -18,18 +18,60 @@ def items(self, query):
         return query.paginate(self.page, self.page_size, False).items
 
 
-def standardize_response(data, errors, status, status_code=200):
-    resp = {
-        "status": status,
-        "apiVersion": API_VERSION
+def standardize_response(payload, status_code=200):
```

Comment:
What do you think about making the default for payload `None`, so it would look like 
```python
def standardize_response(payload=None, status_code=200)
```
Then, when you don't have a payload (e.g., when doing a 401 unauthorized), you can just say `standardize_response(status_code=401)`?